src/iptc.cpp

```diff
@@ -79,6 +80,90 @@ namespace {
         uint16_t record_;
 
     }; // class FindIptcdatum
+
+    bool is_utf8(const std::string& string)
```

Comment:
Hm, this is slightly modified version of the code posted on stackoverflow. Don't know if we can simply copy & paste it here for copyright reasons.

Reply:
Yes, I just wanted to show in the PR that in this way it is possible to deal with non UTF-8 values. I will probably drop this implementation and write something from scratch now that I understand better. I'll do that if both of you agree with the idea behind the PR.